lang/it/toucan.po

```diff
@@ -2,9 +2,9 @@ msgid ""
 msgstr ""
 "Project-Id-Version: Toucan\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2016-06-19 15:42+0200\n"
+"POT-Creation-Date: 2018-05-04 14:45+0200\n"
 "PO-Revision-Date: \n"
-"Last-Translator: Alex Tereschenko <alttoucan@gmail.com>\n"
+"Last-Translator: Giacomo Margarito <alttoucan@gmail.com>\n"
```

Comment:
This line needs email address update as well - could you please either do that or just let me know what your address should be - and I'll correct it by myself when merging?